polaris-react/src/components/Page/components/Header/Header.tsx

```diff
@@ -93,6 +97,8 @@ export function Header({
   const i18n = useI18n();
   const {polarisSummerEditions2023} = useFeatures();
   const {isNavigationCollapsed} = useMediaQuery();
+  const {mode} = useIndexFiltersManager();
+  const shouldDisableActions = mode !== IndexFiltersMode.Default;
```

Comment:
nit: Is the word "should" providing value here, should this just be `disableActions`?